invenio_app_rdm/theme/assets/js/invenio_app_rdm/search/config/index.js

```diff
@@ -0,0 +1,68 @@
+export { templates } from "./templates";
+
+const aggregations = [
+  {
+    title: "Access Right",
+    agg: {
+      field: "access_right",
+      aggName: "access_right"
+    }
+  },
+  {
+    title: "Types",
+    agg: {
+      field: "resource_type",
+      aggName: "resource_type"
+    }
+  }
+];
+
+const sortValues = [
+  {
+    text: "Best match",
+    sortBy: "bestmatch",
+    sortOrder: "desc",
+    defaultOnEmptyString: true
+  },
+  {
+    text: "Newest",
+    sortBy: "mostrecent",
+    sortOrder: "asc",
+    default: true
+  },
+  {
+    text: "Oldest",
+    sortBy: "mostrecent",
+    sortOrder: "desc"
+  }
+];
+
+const resultsPerPageValues = [
+  {
+    text: "10",
+    value: 10
+  },
+  {
+    text: "20",
+    value: 20
+  },
+  {
+    text: "50",
+    value: 50
+  }
+];
+
+const logo = "/static/images/logo.svg";
+
+const searchApi = {
+  url: "https://localhost:5000/api/records/",
```

Comment:
Does this work also in case of containerized? (Wondering about ports, etc.) tho it should since uwsgi will still run in 5000.

Reply:
It should as everything run in the same port. But yes this should come from the backend in the longer run!